bindings/python/algorithm/expose-contact-dynamics.cpp

```diff
@@ -25,20 +25,41 @@ namespace pinocchio
     
     BOOST_PYTHON_FUNCTION_OVERLOADS(forwardDynamics_overloads, forwardDynamics_proxy, 7, 9)
 
-    // TODO: overloading impulseDynamics directly, as done for forwardDynamics, was apparently not working (it crashed for 5 arguments)
-    // Therefore, it was necessary to resort to a proxy
+//    // TODO: overloading impulseDynamics directly, as done for forwardDynamics, was apparently not working (it crashed for 5 arguments)
```

Comment:
This is normal. You cannot use BOOST_PYTHON_FUNCTION_OVERLOADS here. BOOST_PYTHON_FUNCTION_OVERLOADS only works when you are overloading the SAME function with optional arguments. But here you have to completely different signatures.
Example:
```
foo(double x, double y=0., int z=0.)
```
--> You can use BOOST_PYTHON_FUNCTION_OVERLOADS to have `foo(x)`, `foo(x,y)` and `foo(x,y,z)` in Python
```
foo(double x, int y)
foo(int y, double x)
```
--> You cannot use BOOST_PYTHON_FUNCTION_OVERLOADS

Reply:
On the other hand, if you have
```
foo(double x, double y=0., int z=0.)
foo(int w, double a=0., int b=0.)
```
I am not sure whether you can use BOOST_PYTHON_OVERLOADS separately on the first one and on the second one. My intuition is yes

Reply:
In fact, these are old commented lines. I need to remove them.